engines/aws/app/models/aws/cli.rb

```diff
@@ -33,6 +33,16 @@ def execute(*args)
       )
     end
 
+    def get_description(args, find_message, not_found_message)
```

Comment:
Let's make the arguments more obvious - how about `not_found_exception` instead of `find_message`, and `response` or `not_found_response` instead of `not_found_message`

Reply:
Done